### ISSUE TITLE
Nested Folders/Boards!!!!!

### ISSUE THREAD

**Issue by aminadbul** [rivafarabi/deckboard#360]
Please create this feature, it will be just like the elgato streamdeck. #

**Comment by edos145** [comment]
can't you do this with "switch board" or is there something I'm missing?

**Comment by Yurlyn** [comment]
No, you got it right, Edos145. Use the Switch Board button. This is what my DB looks like
![image](https://user-images.githubusercontent.com/42919979/74962073-79078500-540f-11ea-917b-75261c1fe43c.png)
